docs/.vuepress/containers/examples/examples.js

```diff
@@ -7,6 +7,19 @@ const EXAMPLE_REGEX = /^(example)\s*(#\S*|)\s*(\.\S*|)\s*(:\S*|)\s*([\S|\s]*)$/;
 
 const { buildCode } = require('./code-builder');
 const { jsfiddle } = require('./jsfiddle');
+const {
+  getDefaultFramework,
+  isEnvDev
+} = require('../../helpers');
+const {
+  getContainerFrontMatterLength,
+  getContainerFramework
+} = require('../helpers');
+const {
+  SnippetTransformer,
+  logChange,
+  SUPPORTED_FRAMEWORKS
+} = require('../../tools/snippet-transform/snippetTransformer');
```

Comment:
Wouldn't it be consistent to rename the folder from `snippet-transform` to `snippet-transformer`?

Reply:
I was trying to be consistent with the other folders in `./.vuepress/tools` - `jsdoc-convert` and `version`, but renaming it to `snippet-transformer` sounds good to me 👍